packages/wmr/index.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * @type {typeof import("wmr").defineConfig}
+ */
+export function defineConfig(config) {
+	return async options => (typeof config === 'function' ? config(options) : config);
```

Comment:
I ran into the function-or-object thing when porting the Glitch demo from Vite to WMR, and I'm thinking maybe it makes sense for us to do this step in `normalize-options.js`, to keep `defineConfig()` as a pure pass-through for TS purposes. Thoughts?

Reply:
Good idea, that's more elegant than doing it here. `defineCinfig` should do nothing in the best case